sisl/atom.py

```diff
@@ -1644,6 +1647,9 @@ def __str__(self):
             s += ' {1}: {0},\n'.format(len(idx), str(a).replace('\n', '\n '))
         return s + '}'
 
+    def __repr__(self):
+        return f"<sisl.{self.__class__.__name__} nspecies={len(self._atom)}, na={len(self)}, no={self.no}>"
```

Comment:
Instead of doing writing `<sisl.{self.__...` I think we should write: `<{self.__module__}.{self.__class__.__name}` since in that way we can alter the module level and have everything separated and clear :)

I had planned on cleaning up module classes like this for some time. This seems like a good start for its use.